helpers/argumentParsers.go

```diff
@@ -2969,3 +2969,317 @@ func ParseIoUringRequestFlags(rawValue uint64) IoUringRequestFlag {
 
 	return IoUringRequestFlag{stringValue: strings.Join(f, "|"), rawValue: uint32(rawValue)}
 }
+
+// =====================================================
+
+// GupFlag represents GUP (Get User Pages) flags since version 6.3
+type GupFlag struct {
```

Comment:
golang nit: `GUPFlag` as the letters are initials of different words. P.S.: Hopefully it's not the case of the ID != Id conundrum 😄.